examples/cms-umbraco-heartcore/README.md

```diff
@@ -0,0 +1,148 @@
+# A statically generated blog example using Next.js and Umbraco Heartcore
+
+This example showcases Next.js's [Static Generation](https://nextjs.org/docs/basic-features/pages) feature using [Umbraco Heartcore](https://umbraco.com/heartcore) as the data source.
+
+## Demo
+
+- **Live**: [https://next-blog-umbraco-heartcore.vercel.app/](https://next-blog-umbraco-heartcore.vercel.app/)
+- **Preview Mode**: [https://next-blog-umbraco-heartcore.vercel.app/api/preview...](https://next-blog-umbraco-heartcore.vercel.app/api/preview?secret=ivfgc53v8d8ikh7z4nnd5hzwgs298ma6&slug=/posts/learn-how-to-pre-render-pages-using-static-generation-with-next-js)
```

Comment:
Is it okay to include this?

Reply:
Should be fine as it only gives access to preview, I looked at the GraphCMS example and theres also have a link to preview mode